docs/guides/serverless-hamsim-template.ipynb

```diff
@@ -0,0 +1,1033 @@
+{
```

Comment:
<p>I'd like to explicitly call out the logging aspect of this template. Could we put the above print statement into its own cell and add the following text to explain it?</p><p> </p><blockquote>When the function template is running, it is helpful to return information in the logs, using print statements, so that you can better evaluate the progress of your workload. Here is a simple example of printing the <code>estimator_options</code> so that there is a record of the actual Estimator options used. There are many more example like this throughout the program to report back progress during execution. Including the value of the objective function during the iterative component of AQC-Tensor, or the two-qubit depth of the final ISA circuit intended for execution on hardware.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>